packages/kbn-dev-utils/src/run/run.ts

```diff
@@ -75,4 +79,6 @@ export async function run(fn: RunFn, options: RunOptions = {}) {
   } finally {
     cleanup.execute();
   }
+
+  await metrics.reportSuccess();
```

Comment:
Do we want to report errors here too?

Reply:
a03610afdb130072e1c60b082e4d480ed2d3653d

Reply:
Sorry, what I meant was that we're not reporting errors

Reply:
We should keep success here I think as it is after cleanup

Reply:
Oh, you mean to collect the actual error? Yeah, we can do that.

Reply:
I figure it makes sense since we're doing it in run_with_commands

Reply:
2a7667e01d1586d5b5f7ab60cdf6f011b5ec1c7c